src/EFCore.PG.NodaTime/Storage/Internal/NpgsqlNodaTimeTypeMappingSourcePlugin.cs

```diff
@@ -0,0 +1,199 @@
+            _dateRange = new NpgsqlRangeTypeMapping("daterange", typeof(NpgsqlRange<LocalDate>), _date);
+
+            var storeTypeMappings = new Dictionary<string, RelationalTypeMapping[]>(StringComparer.OrdinalIgnoreCase)
+            {
+                { "timestamp", new RelationalTypeMapping[] { _timestampInstant, _timestampLocalDateTime } },
+                { "timestamp without time zone", new RelationalTypeMapping[] { _timestampInstant, _timestampLocalDateTime } },
+                { "timestamptz", new RelationalTypeMapping[] { _timestamptzInstant, _timestamptzZonedDateTime, _timestamptzOffsetDateTime } },
+                { "timestamp with time zone", new RelationalTypeMapping[] { _timestamptzInstant, _timestamptzZonedDateTime, _timestamptzOffsetDateTime } },
+                { "date", new RelationalTypeMapping[] { _date } },
+                { "time", new RelationalTypeMapping[] { _time } },
+                { "time without time zone", new RelationalTypeMapping[] { _time } },
+                { "timetz", new RelationalTypeMapping[] { _timetz } },
+                { "time with time zone", new RelationalTypeMapping[] { _timetz } },
+                { "interval", new RelationalTypeMapping[] { _period } },
+
+                { "tsrange", new RelationalTypeMapping[] { _timestampInstantRange, _timestampLocalDateTimeRange } },
+                { "tstzrange", new RelationalTypeMapping[] { _timestamptzInstantRange, _timestamptzZonedDateTimeRange, _timestamptzOffsetDateTimeRange } },
+                { "daterange", new RelationalTypeMapping[] { _dateRange} }
+            };
+
+            var clrTypeMappings = new Dictionary<Type, RelationalTypeMapping>
+            {
+                { typeof(Instant), _timestampInstant },
+                { typeof(LocalDateTime), _timestampLocalDateTime },
+                { typeof(ZonedDateTime), _timestamptzZonedDateTime },
+                { typeof(OffsetDateTime), _timestamptzOffsetDateTime },
+                { typeof(LocalDate), _date },
+                { typeof(LocalTime), _time },
+                { typeof(OffsetTime), _timetz },
+                { typeof(Period), _period },
+
+                { typeof(NpgsqlRange<Instant>), _timestampInstantRange },
+                { typeof(NpgsqlRange<LocalDateTime>), _timestampLocalDateTimeRange },
+                { typeof(NpgsqlRange<ZonedDateTime>), _timestamptzZonedDateTimeRange },
+                { typeof(NpgsqlRange<LocalDate>), _dateRange },
+                { typeof(NpgsqlRange<OffsetDateTime>), _timestamptzOffsetDateTimeRange }
+            };
+
+            StoreTypeMappings = new ConcurrentDictionary<string, RelationalTypeMapping[]>(storeTypeMappings, StringComparer.OrdinalIgnoreCase);
+            ClrTypeMappings = new ConcurrentDictionary<Type, RelationalTypeMapping>(clrTypeMappings);
+        }
+
+        public RelationalTypeMapping FindMapping(in RelationalTypeMappingInfo mappingInfo) =>
```

Comment:
nit: It seems like the majority of our expression-bodied assignments push `=>` to the line following the signature in multiline cases.

Reply:
I think it depends whether the expression body contains multiple lines that should be aligned. Regardless, in this case I just move everything to a single line after the signature, as it's short enough...